documentation/asciidoc/accessories/touch-display-2/about.adoc

```diff
@@ -127,3 +127,23 @@ To specify these options, add them, separated by commas, to your `dtoverlay` lin
 ----
 dtoverlay=vc4-kms-dsi-ili9881-7inch,sizex=400,invx,invy
 ----
+
+=== Installation and software setup on Compute Module based devices.
```

Comment:
Copy/paste the first paragraph into https://github.com/raspberrypi/documentation/blob/develop/documentation/asciidoc/accessories/display/display_intro.adoc as well?

Reply:
Done